content/github/importing-your-projects-to-github/adding-an-existing-project-to-github-using-the-command-line.md

```diff
@@ -43,7 +43,7 @@ versions:
 	![Copy remote repository URL field](/assets/images/help/repository/copy-remote-repository-url-quick-setup.png)
 8. In Terminal, [add the URL for the remote repository](/articles/adding-a-remote) where your local repository will be pushed.
   ```shell
-  $ git remote add origin <em>remote repository URL</em>
+  $ git remote add origin <em>remote_repository_URL</em>
```

Comment:
I wonder if it might be even clearer to follow the format used in: [Creating remotes](https://docs-2850--patch-1.herokuapp.com/en/free-pro-team@latest/github/using-git/about-remote-repositories#creating-remotes)?

That is:

```suggestion
  $ git remote add origin <em> &lt;REMOTE_URL> </em>
```